packages/frontend/src/components/QrReader/index.tsx

```diff
@@ -158,47 +82,18 @@ export default function QrReader({ onError, onScan }: Props) {
       } else {
         onError(error.toString())
       }
-
-      setError(true)
```

Comment:
Yes but that resulted in showing the camera_access_failed error in the UI which I consider wrong since it's a generic errorHandler (for example after an error in handleFileInputChange)